docs/install/bukkit-server-world-creation.rst

```diff
@@ -0,0 +1,88 @@
+=====================
+Bukkit World Creation
+=====================
+
+.. note::
+
+    This guide is intended for the **Bukkit** version of Terra.
+    
+    :doc:`Other Platform Installations <index>` :octicon:`chevron-right`
+
+.. warning::
+    We will be replacing the server's default world with a new Terra world.
+
+    Because we are working with changes to worlds **ensure that you have made the necessary backups** before making any destructive changes!
+    
+
+We will be replacing the server's default world with a new one that uses Terra as its generator.
+
+If you would like to use a world manager like Multiverse Core to create a world instead of manually setting it as
+outlined here, please refer to :doc:`Setting a Generator Through a World Manager<worldmanager-bukkit-world-creation>` :octicon:`chevron-right`
+
+Procedure
+---------
+
+#. Ensure your server is not running.
+
+#. If you missed it above, please make a backup of any relevant world folders in your server directory.
+
+    If you're using a fresh server you won't need to worry about this!
+
+#. Configure your server's world to use the new config as a generator:
+
+   #. Navigate to the `bukkit.yml` file which is also contained within your server directory, and open it with any text editor.
+
+   #. Assign your new generator to the default world by **adding the following lines to the end of the file**:

```

Comment:
might be wise to add a .. warning after this explaining that these lines are not present in the file by default, just to make it that much harder to miss.